source/testing.rst

```diff
@@ -0,0 +1,57 @@
+.. _testing:
+
+Testing
+=======
+
+DHIL Symfony applications use `PHPUnit Testing Framework`_ to run automated tests on the code components. PHPUnit manual is `here <https://phpunit.de/manual/6.3/en/index.html>`_. A comprehensive tutorial can be found `here <https://jtreminio.com/2013/03/>`_.
+
+.. note:: PHPUnit is enabled after installing Composer and CEWW specific composer dependencies. It should be located at ceww/vendor/bin/phpunit. If, for some reason, this is not the case, simply follow the installation guide `here <https://phpunit.de/manual/current/en/installation.html>`_. Make sure you use the right version (supporting PHP 5.5.9 or later).
+
```

Comment:
If composer install didn't get the right version of phpunit, there's a bigger problem with composer that needs to be addressed.

Reply:
I just don't mention it then : ) fixed!